packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -62,14 +57,6 @@ const testContainerConfig: ITestContainerConfig = {
 
 const usageErrorMessage = "Empty file summary creation isn't supported in this driver.";
 
-const containerCloseAndDisposeUsageErrors = [
```

Comment:
Is this being removed because the container not attaching isn't necessarily fatal anymore?

Reply:
exactly, the container doesn't close, just attach throws errors.